### PR TITLE
Add bssid to connect, findAndConnect and registerWifiNetwork

### PR DESCRIPTION
**Pull request by fernando-s97**
This is my first PR. If I'm making something wrong, please tell me

### CODE REVIEW

File: android/src/main/java/info/whitebyte/hotspotmanager/ClientScanResult.java [hunk 1]
```diff
@@ -17,48 +17,48 @@
 package info.whitebyte.hotspotmanager;
 
 public class ClientScanResult {
-  private String IpAddr;
```
Comment (diegotori) [inline android/src/main/java/info/whitebyte/hotspotmanager/ClientScanResult.java:20]: You may wanna revert the formatting. Don't think its necessary to make this change.

File: android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java [hunk 2]
```diff
@@ -104,77 +102,55 @@ private void cleanup() {
         moWiFi = null;
         moWiFiAPManager = null;
     }
-
```
Comment (diegotori) [inline android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java:107]: Why is V1 embedding being removed? It should remain for backwards compatibility.

**Comment by diegotori** [comment]
Also, this should also be added to iOS in order to maintain parity. However, if that is not possible, then it should be well documented on the Dart frontend.

**Comment by diegotori** [comment]
@daadu any thoughts on the above with regards to the dev having to also add bssid support for iOS (or at least document it as to why it's not supported on that platform)?

**Comment by daadu** [comment]
Closing this, since similar efforts are made with #217

**Pull request closed without merging**